core/src/test/java/org/apache/iceberg/TestRemoveSnapshots.java

```diff
@@ -134,22 +135,24 @@ public void testExpireOlderThanWithDelete() {
     assertThat(table.snapshot(firstSnapshot.snapshotId())).isNull();
     assertThat(table.snapshot(secondSnapshot.snapshotId())).isNull();
 
+    Set<String> expectedDeletedFiles =
```

Comment:
Could we revert some of these unrelated test changes/refactorings, just to make the diff smaller and easier to review? Not opposed to the refactoring itself, but feel like that should be separate.

Reply:
Sounds good, reverted all unnecessary changes, will have a separate PR to refactor TestRemoveSnapshots later